src/sqladmin-fetcher.ts

```diff
@@ -149,8 +150,24 @@ export class SQLAdminFetcher {
     // Resolve dnsName into IP address for PSC enabled instances.
     // Note that we have to check for PSC enablement because CAS instances
     // also set the dnsName field.
-    if (dnsName && pscEnabled) {
-      ipAddresses.psc = dnsName;
+    if (pscEnabled) {
```

Comment:
are we still supposed to check for pscEnabled?

Reply:
Updated. I think we are checking if pscEnabled but only for the `dnsName` field.